draft-ietf-tls-dtls-rrc.md

```diff
@@ -142,18 +143,186 @@ struct {
     select (return_routability_check.msg_type) {
         case path_challenge: Cookie;
         case path_response:  Cookie;
+        case path_delete:  Cookie;
     };
 } return_routability_check;
 ~~~~
 
-The newly introduced `return_routability_check` message contains a cookie.  The
-cookie is a 8-byte field containing arbitrary data.
+The cookie is a 8-byte field containing arbitrary data.
 
 The `return_routability_check` message MUST be authenticated and encrypted using
 the currently active security context.
 
+# Off-Path Packet Forwarding {#off-path}
+
+An off-path attacker that can observe packets might forward copies of
+genuine packets to endpoints. If the copied packet arrives before
+the genuine packet, this will appear as a NAT rebinding. Any genuine
+packet will be discarded as a duplicate. If the attacker is able to
+continue forwarding packets, it might be able to cause migration to a
+path via the attacker. This places the attacker on-path, giving it
+the ability to observe or drop all subsequent packets.
+
+This style of attack relies on the attacker using a path that has
+approximately the same characteristics as the direct path between
+endpoints. The attack is more reliable if relatively few packets are
+sent or if packet loss coincides with the attempted attack.
+
+A data packet received on the original path that increases the
+maximum received packet number will cause the endpoint to move back
+to that path. Eliciting packets on this path increases the
+likelihood that the attack is unsuccessful.
+
+{{fig-off-path}} demonstrates the case where a receiver receives a
+packet with a new source IP address and/or new port number. The
+receiver needs to determine whether this path change is caused
+by an attacker and will sent a RRC message with a path_challenge
+along the old path. 
+
+~~~~
+        new   +--------+  old
+        path  |        |  path
+       +----->|Receiver|<-----+
+       |      |        |      |
+       |      +--------+      |
+       |                      |
+       |                      |
+       |                      |
+       |                      |
+       |                      |
+ +----------+                 |
+ | Attacker?|                 |
+ +----------+                 |
+       |                      |
+       |                      |
+       |                      |
+       |      +--------+      |
+       |      |        |      |
+       +------| Sender |------+
+              |        |
+              +--------+
+~~~~
+{: #fig-off-path title="Off-Path Packet Forwarding Scenario"}
+
+Three cases need to be considered:
+
+Case 1: The old path is dead, which leads to a timeout of RRC-1. 
+
+As shown in {{fig-old-path-dead}}, RRC-2 needs to be sent on the
+new path. In this situation the switch to the new path is
+considered legitimate (assuming that the attacker is unable to
```

Comment:
I'd remove the parenthetical because this is the overall assumption not something that is specific to this situation.